Unable to create a pip whl.

What related GitHub issues or StackOverflow threads have you found by searching the web for your problem?
#2040
Environment info
Installed version of CUDA and cuDNN:
(please attach the output of ls -l /path/to/cuda/lib/libcud*):
-rw-r--r-- 1 root root  189170 Dec 23  2015 /usr/local/cuda/lib/libcudadevrt.a
lrwxrwxrwx 1 root root      16 Dec 23  2015 /usr/local/cuda/lib/libcudart.so -> libcudart.so.7.5
lrwxrwxrwx 1 root root      19 Dec 23  2015 /usr/local/cuda/lib/libcudart.so.7.5 -> libcudart.so.7.5.18
-rwxr-xr-x 1 root root  311596 Dec 23  2015 /usr/local/cuda/lib/libcudart.so.7.5.18
-rw-r--r-- 1 root root  558020 Dec 23  2015 /usr/local/cuda/lib/libcudart_static.a
lrwxrwxrwx 1 root root      17 Dec 23  2015 /usr/local/cuda/lib/libcuinj32.so -> libcuinj32.so.7.5
lrwxrwxrwx 1 root root      20 Dec 23  2015 /usr/local/cuda/lib/libcuinj32.so.7.5 -> libcuinj32.so.7.5.18
-rwxr-xr-x 1 root root 5396088 Dec 23  2015 /usr/local/cuda/lib/libcuinj32.so.7.5.18
If installed from source, provide

The commit hash (git rev-parse HEAD)

9b69ec3

The output of bazel version

Build target: bazel-out/local-fastbuild/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
Build time: Thu Jan 01 00:00:00 1970 (0)
Build timestamp: Thu Jan 01 00:00:00 1970 (0)
Build timestamp as int: 0
If possible, provide a minimal reproducible example (We usually don't have time to read hundreds of lines of your code)
Quite simply, I am going through the instructions here(https://www.tensorflow.org/versions/r0.10/get_started/os_setup.html#create-the-pip-package-and-install), and get the following error:
tc:/deeplearning/tensorflow$ bazel-bin/tensorflow/tools/pip_package/build_pip_package.sh /tmp/tensorflow_pkg
bash: bazel-bin/tensorflow/tools/pip_package/build_pip_package.sh: No such file or directory
What other attempted solutions have you tried?
The fix from here (#2040), which was to do: cp -r bazel-bin/tensorflow/tools/pip_package/build_pip_package.runfiles/main/* bazel-bin/tensorflow/tools/pip_package/build_pip_package.runfiles/
seemed to run properly, but this did not work in the end.